yang-module-versioning/draft-ietf-netmod-yang-module-versioning.txt

```diff
@@ -1768,11 +1768,97 @@ Internet-Draft    Updated YANG Module Revision Handling     October 2022
    Bjorklund, Italo Busi, Tom Hill, Scott Mansfield, Kent Watsen for
    their contributions and review comments.
 
-10.  Security Considerations
+10.  Security considerations
+
+10.1.  Security considerations for module revisions
+
+   As discussed in the introduction of this document, YANG modules
```

Comment:
YANG modules and submodules?

Reply:
I looked through the new section and I *think* most of the uses of "module" in here are "good enough" because changes to a submodule also means the overall module has changed. But I will add "or submodule" in a few places.